panther_manager/test/plugins/test_call_set_bool_service_node.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstdint>
+#include <string>
+
+#include <behaviortree_cpp/bt_factory.h>
+#include <gtest/gtest.h>
+#include <rclcpp/rclcpp.hpp>
+
+#include <panther_manager/plugins/action/call_set_bool_service_node.hpp>
+
+#include <panther_manager_plugin_test_utils.hpp>
+
+void ServiceFailedCallback(
+  const std_srvs::srv::SetBool::Request::SharedPtr request,
+  std_srvs::srv::SetBool::Response::SharedPtr response)
+{
+  response->message = "Failed callback pass!";
+  response->success = false;
+  RCLCPP_INFO_STREAM(
+    rclcpp::get_logger("test_set_bool_plugin"), response->message << " data: " << request->data);
+}
+
+void ServiceSuccessCallbackCheckTrueValue(
+  const std_srvs::srv::SetBool::Request::SharedPtr request,
+  std_srvs::srv::SetBool::Response::SharedPtr response)
+{
+  response->message = "Successfully callback pass!";
+  response->success = true;
+  RCLCPP_INFO_STREAM(
+    rclcpp::get_logger("test_set_bool_plugin"), response->message << " data: " << request->data);
+
+  EXPECT_EQ(request->data, true);
+}
+
+void ServiceSuccessCallbackCheckFalseValue(
+  const std_srvs::srv::SetBool::Request::SharedPtr request,
+  std_srvs::srv::SetBool::Response::SharedPtr response)
+{
+  response->message = "Successfully callback pass!";
+  response->success = true;
+  RCLCPP_INFO_STREAM(
+    rclcpp::get_logger("test_set_bool_plugin"), response->message << " data: " << request->data);
+
+  EXPECT_EQ(request->data, false);
+}
+
+TEST(TestCallSetBoolService, good_loading_call_set_bool_service_plugin)
```

Comment:
```suggestion
TEST(TestCallSetBoolService, GoodLoadingCallSetBoolServicePlugin)
```
We are using CamelCase for test names